conda-recipes/llvmdev/meta.yaml

```diff
@@ -54,7 +54,7 @@ requirements:
   host:
     # needed for llc at runtime
     - zlib # [not win]
-    - xar # [osx]
+    - xar # [osx and x86_64]
```

Comment:
xref to previous discussion establishing that this is not needed: https://github.com/numba/llvmlite/pull/813/files#r826859213

Reply:
Was this missed by the original PR and do we need an additional PR for `main` to make this change?